upup/pkg/fi/cloudup/networking.go

```diff
@@ -83,18 +84,13 @@ func usesCNI(c *api.Cluster) bool {
 	return true
 }
 
-// TODO: we really need to sort this out:
```

Comment:
They answered the issue upstream.  The CNI plugin is versioned when the k8s release.  That is how I determined that we do not need a new version for 1.7.0 k8s.